enterprise/settings.mdx

```diff
@@ -128,3 +128,23 @@ DEFAULT_ORG_ROLE=view_all
 ```
 
 Note you can run an [Enterprise self-check](/docs/enterprise/troubleshooting) to verify the connection and lookup LDAP information, but some configuration errors are only visible once logging in.
+
+## Sidekiq monitoring using Prometheus
```

Comment:
Maybe "Worker queue monitoring using Prometheus"?

(the fact that we're using Sidekiq is an implementation detail in a sense)

Reply:
```suggestion
## Worker queue monitoring using Prometheus
```